asset-schemas/ontology_info_schema.json

```diff
@@ -41,6 +41,15 @@
             },
             "PATO": {
               "$ref": "#/definitions/ontologyEntry"
+            },
```

Comment:
we should probably remove this from the schema since its already pretty well-documented elsewhere, but we should do that in main rather than in this experimental branch